artifacts/CAMARA_common.yaml

```diff
@@ -265,6 +265,16 @@ components:
       status: 501
       code: NOT_IMPLEMENTED
       message: This functionality is not implemented yet
+  NotSupported:

```

Comment:
```suggestion
  PhoneNumberNotSupported:

```

Reply:
@PedroDiez 
Suggest not to be specific to "phoneNumber", so this error can be reused.